sklearn/linear_model/tests/test_base.py

```diff
@@ -105,12 +105,14 @@ def test_raises_value_error_if_positive_and_sparse():
         reg.fit(X, y)
 
 
-def test_raises_value_error_if_sample_weights_greater_than_1d():
+def test_raises_value_error_if_sample_weights_greater_than_1d(random_state=0):
     # Sample weights must be either scalar or 1D
 
     n_sampless = [2, 3]
```

Comment:
The ending `ss` is not nice :)
I think that we could parametrize the function with something like

```python
@pytest.mark.parametrize("n_samples, n_features", [(2, 3), (3, 2)])
```

This would be much nicer to read.

Reply:
Good idea :) addressed